index.html

```diff
@@ -611,9 +664,9 @@ <h3>Terminology</h3>
         <h3>Data Model</h3>
 
         <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] SHOULD
+A [=controller document=] specifies one or more relationships between
+an [=identifier=] and a set of verification methods and/or service 
+endpoints. The [=controller document=] SHOULD
```

Comment:
```suggestion
a [=subject=], when identified by the [=base identifier=], and a set of
verification methods and/or service  endpoints. The [=controller document=] SHOULD
```

Reply:
See previous comment. The semantics here only definitively relate the identifier to the method. 

The subject may or may not be involved.

Reply:
Either...
```suggestion
an [=identifier=] and a set of [=verification methods=] and/or service 
endpoints. The [=controller document=] SHOULD
```

Reply:
or...
```suggestion
a [=subject=], when identified by the [=base identifier=], and
a set of [=verification methods=] and/or [=service endpoints=].
The [=controller document=] SHOULD
```